tests/js/server/resilience/failover-view/resilience-synchronous-repl-with-arangosearch-view-cluster-grey.js

```diff
@@ -119,21 +120,23 @@ function SynchronousReplicationWithViewSuite () {
     assertTrue(pos >= 0);
     assertTrue(suspendExternal(global.instanceInfo.arangods[pos].pid));
     console.info("Have failed follower", follower);
+    failedState.follower = follower;
   }
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief heal the follower
 ////////////////////////////////////////////////////////////////////////////////
 
-  function healFollower() {
-    var follower = cinfo.shards[shards[0]][1];
+  function healFollower(follower = null) {
+    if (follower == null) follower = cinfo.shards[shards[0]][1];
```

Comment:
This will trigger a jslint warning. Should be:
```suggestion
    if (follower === null) {
      follower = cinfo.shards[shards[0]][1];
    }
```